mkt/features/tests/test_utils.py

```diff
@@ -0,0 +1,59 @@
+from django.test.client import RequestFactory
+
+import mock
+from nose.tools import eq_
+
+import mkt.site.tests
+from mkt.constants.features import FeatureProfile
+from mkt.features.utils import load_feature_profile
+
+
+class TestLoadFeatureProfile(mkt.site.tests.TestCase):
+    def setUp(self):
+        super(TestLoadFeatureProfile, self).setUp()
+        self.profile = FeatureProfile(apps=True)
+        self.signature = self.profile .to_signature()
```

Comment:
Fixed and renamed the file to avoid clashes with `test_utils` package (we no longer use it, but just in case, I'm following the convention we used in the other modules...)